sdv/metadata/table.py

```diff
@@ -661,7 +661,10 @@ def transform(self, data, on_missing_column='error'):
         data = self._transform_constraints(data, on_missing_column)
 
         LOGGER.debug('Transforming table %s', self.name)
-        return self._hyper_transformer.transform(data)
+        try:
+            return self._hyper_transformer.transform(data)
+        except rdt.errors.NotFittedError:
+            return data
```

Comment:
Why do we return `data` if `NotFitted`? Should we not let the error be raised?

Reply:
This happens when data that is all primary keys gets passed. The `HyperTransformer` raises a not fitted error even though it was fit because the `_transformers_sequence` is empty

Reply:
I see. So it seems to rather be a bug on RDT, right?
I think the change is fine, though, because this code is protected by the `if not self.fitted` above, meaning that this `rdt.errors.NotFitted` will only be raised in that scenario that you described.

Reply:
Yeah, I think we will have to fix this in RDT